### PR TITLE
Windows build compatibility via Ruby 2.3.0

### PR DESCRIPTION
**Pull request by damieng**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.2.3
+2.2.4
```
Comment (lee-dohm) [inline .ruby-version]: Did you mean for this to be `2.3.0` also?

Comment (damieng) [inline .ruby-version]: Oops.

**Comment by lee-dohm** [comment]
Works fine! Thanks :bow:

**Comment by lee-dohm** [comment]
It turns out that the auto-publish infrastructure we're using doesn't support Ruby v2.3.0 right now #222 ... reverting to unblock auto-publishing until we can figure out the right solution.

**Pull request merged**